client/components/index.html

```diff
@@ -2,8 +2,8 @@
 <html>
 <head>
 	<title>COMPONENTS</title>
-	<script type="text/javascript" src="../build/components.js"></script>
-	<script type="text/javascript" src="./node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+	<script type="text/javascript" src="../../magenta4live.amxd/code/public/components.js"></script>
+	<script type="text/javascript" src=".../../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
```

Comment:
Just checked [the package](https://github.com/webcomponents/webcomponentsjs) and it seems like it's been deprecated. I also wonder if this is still necessary or not given that CEF probably implements Web Components without needing a polyfill.